tracer/src/Datadog.Trace/ClrProfiler/Instrumentation.cs

```diff
@@ -446,10 +446,29 @@ private static void StartDiagnosticManager()
         }
 #endif
 
-        private static void InitRemoteConfigurationManagement(Tracer tracer)
+        private static void InitLiveDebugger(Tracer tracer)
         {
+            var settings = tracer.Settings;
+            var debuggerSettings = DebuggerSettings.FromDefaultSource();
+
+            if (!settings.IsRemoteConfigurationAvailable)
+            {
+                // live debugger requires RCM, so there's no point trying to initialize it if RCM is not available
+                if (debuggerSettings.Enabled)
+                {
+                    Log.Warning("Live Debugger is enabled but remote configuration is not available in this environment, so live debugger cannot be enabled.");
+                }
+                else
+                {
+                    Log.Information("Live Debugger is disabled. To enable it, please set DD_DYNAMIC_INSTRUMENTATION_ENABLED environment variable to 'true'.");
```

Comment:
I'm not sure if this log line is needed here since enabling the live debugger would simply end up in the previous branch, since remote configuration is not available